pkg/build/pipelines/split/bin.yaml

```diff
@@ -0,0 +1,38 @@
+name: Split executable files
+
+needs:
+  packages:
+    - busybox
+
+inputs:
+  package:
+    description: |
+      The package to split executable files from
+    required: false
+
+pipeline:
+  - runs: |
+      pkgname="${{context.name}}"
+      pkgnobin="${pkgname%%-bin}"
+      pkgnobindir="${{targets.outdir}}/$pkgnobin"
+
+      if [ -n "${{inputs.package}}" ]; then
+        PACKAGE_DIR="${{targets.outdir}}/${{inputs.package}}"
+      elif [ "$pkgname" != "$pkgnobin" ] && [ -d "$pkgnobindir" ]; then
+        PACKAGE_DIR="$pkgnobindir"
+      else
+        PACKAGE_DIR="${{targets.destdir}}"
+      fi
+
+      if [ "$PACKAGE_DIR" == "${{targets.contextdir}}" ]; then
+        echo "ERROR: Package can not split files from itself!"
+        exit 1
+      fi
+
+      dir=usr/bin
+      if [ -e "$PACKAGE_DIR/$dir" ] || [ -L "$PACKAGE_DIR/$dir" ]; then
```

Comment:
`-L` here seems wierd.  it only matters if bin/ is a dangling symlink.  And if it is a symlink.. what does the 'mv' do?

maybe its best to just do something like:
```
if [ ! -d "$PACKAGE_DIR/$dir" ]; then
   echo "$PACKAGE_DIR/$dir: not a directory"
   exit 1
fi

Reply:
yeah, i wondered the same thing - kept it for consistency w/ `split/dev`. I'm forgetting about `split/dev` now.